requirements/dev.txt

```diff
@@ -30,5 +30,5 @@ boto3-stubs[appconfig,serverlessrepo]>=1.19.5,==1.*
 types-PyYAML~=5.4
 types-jsonschema~=3.2
 
-# schema generation, requiring features in >=1.10
+# resource validation & schema generation, requiring features in >=1.10
 pydantic~=1.10
```

Comment:
Shouldn't be in `dev.txt` if code is in `samtranslator`, it's bound to be imported and cause issues in production.